### PR TITLE
Implementing min order

### PR DESCRIPTION
**Pull request by rajlash**

### CODE REVIEW

File: app/models/terminal.rb [hunk 1]
```diff
@@ -22,27 +22,28 @@ class Terminal < ApplicationRecord
...
 
   def ordered_amount
-    Order.where(status: 'pending',terminal_id: self.id).sum(:total_cost)
+    Order.where(status: 'pending',terminal_id: id).sum(:total_cost)
   end
 
   def confirmation_possibility
-    possibility = (100 * self.ordered_amount / self.min_order_amount).round(2)
-    possibility < 100 ? possibility : 100
+    return 100 if min_order_amount ==0
```
Comment (pramodshinde) [inline app/models/terminal.rb]: return 100 if min_order_amount ==0 change this to `return 100 if min_order_amount == 0`. Use space before and after the operators.

File: lib/tasks/scheduler.rake [hunk 2]
```diff
@@ -16,13 +14,25 @@ task :place_order_reminder => :environment do
   end
 end
 
+desc 'This task Titleizes company, terminal, and employee names'
```
Comment (pramodshinde) [inline lib/tasks/scheduler.rake]: titleize MenuItem names as well and add before save for titleize names for all these models.

File: app/models/company.rb [hunk 3]
```diff
@@ -35,7 +35,9 @@ def send_reminders
   end
 
   def top_recommended_terminals
```
Comment (pramodshinde) [inline app/models/company.rb]: Is test cases added for all scenarios for this method?

File: app/models/terminal.rb (hunk 1 above)
Comment (pramodshinde) [reply app/models/terminal.rb]: @rajlash ^^

File: lib/tasks/scheduler.rake (hunk 2 above)
Comment (rajlash) [reply lib/tasks/scheduler.rake]: Done

File: app/models/terminal.rb (hunk 1 above)
Comment (rajlash) [reply app/models/terminal.rb]: Updated the method

**Pull request merged**